packages/split-view/src/SplitView.ts

```diff
@@ -401,6 +401,9 @@ export class SplitView extends SpectrumElement {
 
     protected updated(changed: PropertyValues): void {
         super.updated(changed);
+        if (changed.has('primarySize')) {
+            this.checkResize(true);
```

Comment:
Can't spend too much time digging in, as I'm headed out of town, but you _could_...
```
this.splitterPos = undefined;
this.checkResize();
```
To prevent adding the argument, though it likely disrupts something elsewhere?

Reply:
I'll try that

Reply:
It seems to work too. All tests are passing and changing the value in the storybook redraws the component correctly.

You can pick the preferable variant.